SeaPublicWebsite/Views/Shared/_LanguageControl.cshtml

```diff
@@ -0,0 +1,47 @@
+﻿@using Microsoft.AspNetCore.Http.Extensions
+@using SeaPublicWebsite.Controllers
+@using Microsoft.AspNetCore.Mvc.Localization
+@using System.Globalization
+
+<form method="POST" action="@Url.Action(nameof(EnergyEfficiencyController.LanguageChange), "EnergyEfficiency")">
+    @Html.Hidden("ReturnUrl", Context.Request.GetEncodedPathAndQuery())
+    @Html.AntiForgeryToken()
+    <nav>
+        <div class="sea-language-select govuk-body">
+
+            @Html.AntiForgeryToken()
+
+            <input name="next" type="hidden" value="@ViewContext.HttpContext.Request.Path">
+
+            <ul class="sea-language-select__list">
+                @if (CultureInfo.CurrentCulture.Equals(new CultureInfo("en-GB")))
+                {
+                    <li class="sea-language-select__list-item">
+                        <span aria-current="true">English</span>
+                    </li>
+                    <li class="sea-language-select__list-item">
+                        <input name="language" type="hidden" value="cy">
+                        <button type="submit" lang="cy" rel="alternate" class="govuk-link">
+                            <span class="govuk-visually-hidden">Newid yr iaith ir Gymraeg</span>
```

Comment:
Maybe it's necessary, bit it seems a little odd that the one place we aren't using resource files for translation is on the language control.

Reply:
The way it's structured now, I believe this is a good solution, given we want the text shown to be a fixed language at all times. (Show English always in English, and Welsh always in Welsh, so users of that language can identify it.)

Reply:
I misread this bit, it makes more sense the second time around :)